.github/workflows/docs-pages@v2.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Publish docs to GitHub Pages"
```

Comment:
@dgarcia360 can you please add a doc of this yml?
If we will have this in all projects, better put it in the theme wiki

Reply:
See https://sphinx-theme.scylladb.com/stable/github-pages#available-workflows